index.html

```diff
@@ -0,0 +1,289 @@
+						<li>html</li>
+						<li>css</li>
+						<li>javaScript</li>
+					</ul>
+					<button id="3" class="project-button button" type="button">See Project</button>
+				</div>
+		</section>
+	</div>
+	</div>
+	<div id="about" class="about-me">
+		<section class="written-info">
+			<section>
+				<h2 class="about-me-heading font-style">About <br> Myself</h2>
+				<p class="description font-style">
+					I’m a software developer! I can help you build a product , feature or
+					website Look through some of my work and experience! If you like what
+					you see and have a project you need coded, don’t hestiate to contact
+					me.
+				</p>
+			</section>
+
+			<section>
+				<h4 class="connect font-style">LET'S CONNECT</h4>
+				<ul class="socials">
+					<li>
+						<a class="social_medias twitter" href="https://twitter.com/FentahunAgmas">
+							<img src="images/Vector (1).svg" alt="twitter logo">
+						</a>
+					</li>
+
+					<li>
+						<a class="social_medias" href="https://www.linkedin.com/in/fentahun-agmas-5a01aa162/">
+							<img src="images/Linkedin icon.svg" alt="Linkedin logo">
+						</a>
+					</li>
+					<li>
+						<a class="social_medias" href="">
+							<img src="images/mediumLogo.png" alt="Medium icon" class="medium" />
+						</a>
+					</li>
+					<li>
+						<a class="social_medias" href="https://github.com/Fentahunn">
+							<img src="images/githublogo.png" alt="Github icon" />
+						</a>
+					</li>
+					<li>
+						<a href="#"><img src="images/Vector (2).svg" alt="two fingers"></a>
+					</li>
+				</ul>
+				<button type="button" class="resume-button button">Get my resume</button>
+			</section>
+		</section>
+		<div class="languages_for_web">
+			<section class="languages">
+				<a href="#" class="languages-heading">
+					<h3 class="font-style">Languages</h3>
+					<img class="vector" src="images/arrow-down.png" alt="arrow down" />
+				</a>
+				<ul class="lang">
+					<li>
+						<a class="language" href="#lang">
+							<img src="images/js.svg" alt="javascript logo" />
+							<h4 class="language-text javascript font-style">
+								JavaScript
+							</h4>
+						</a>
+					</li>
+					<li>
+						<a class="language m-left" href="#lang">
+							<img src="images/html.svg" alt="html logo" />
+							<h4 class="language-text html font-style">HTML</h4>
+						</a>
+					</li>
+					<li>
+						<a class="language m-left" href="#lang">
+							<img src="images/css.svg" alt="css logo" />
+							<h4 class="language-text html font-style">CSS</h4>
+						</a>
+					</li>
+				</ul>
+			</section>
+			<section class="frameworks">
+				<div class="framework-list">
+					<h3 class="font-style">Frameworks</h3>
+					<img class="vector" src="images/right-arrow.png" alt="arrow right" />
+				</div>
+				<div class="framework-list">
+					<h3 class="font-style">Skills</h3>
+					<img class="vector" src="images/right-arrow.png" alt="arrow right" />
+				</div>
+			</section>
+		</div>

```

Comment:
kindly, make sure only the main container is a `section` tag.
Please, remove all `sections` tags inside the `div` tags for good semantics reason and best practices. 😄